src/main/resources/i18n/messages.properties

```
@@ -178,6 +178,13 @@ artemisApp.singleUserNotification.title.vcsAccessTokenAdded = New VCS access tok
 artemisApp.singleUserNotification.title.vcsAccessTokenExpired = VCS access token has expired
 artemisApp.singleUserNotification.title.vcsAccessTokenExpiresSoon = VCS access expires soon
 
+email.notification.login.title = A new login on Artemis
+email.notification.login.text = A new login on Artemis was detected. If you are not aware of this, make sure to reset your login credentials
```

I think we should be more direct to the user here
```suggestion
email.notification.login.text = A new login on Artemis was detected. If you have not logged in to Artemis, make sure to reset your login credentials
```

---

Changed in this [commit](https://github.com/ls1intum/Artemis/pull/10770/commits/57c0c1d189d38905805516dcc60359a9f65e9eb2).